lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -120,6 +126,23 @@ export default class BitcoinProcessor {
   public async initialize () {
     console.debug('Initializing TransactionStore');
     await this.transactionStore.initialize();
+    const address = this.privateKey.toAddress();
+    console.debug(`Checking if bitcoin contains a wallet for ${address}`);
+    if (!await this.walletExists(address.toString())) {
+      console.debug(`Configuring bitcoin peer to Watch address ${this.privateKey.toAddress()}. This can take up to 10 minutes.`);
+      const request = {
+        method: 'importpubkey',
+        params: [
+          this.privateKey.toPublicKey().toBuffer().toString('hex'),
+          'sidetree',
+          true
+        ]
+      };
+      await this.rpcCall(request, undefined, false);
```

Comment:
Sanity: so this RPC call blocks up to 10 minutes before returning without timing out? Is there a way to poll (thus print) the status?

Reply:
So the actual command is quick if you don't care about history, but I gave the "rescan" flag, so it hangs and doesn't return until that is complete, which takes about 10 minutes. Theres no progress report that I found aside from whats printed on the bitcoin daemon.